pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -1361,6 +1374,50 @@ static pj_status_t dtls_on_recv(pjmedia_transport *tp, unsigned idx,
         (ds->setup == DTLS_SETUP_ACTPASS || ds->setup == DTLS_SETUP_PASSIVE))
     {
         pj_status_t status;
+
+#if defined(PJMEDIA_SRTP_DTLS_CHECK_HELLO_ADDR) && \
+            PJMEDIA_SRTP_DTLS_CHECK_HELLO_ADDR==1
+
+        /* Check the souce address with the specified remote address from 
+         * the SDP. When ICE is used, the source address checking will be 
+         * done in ICE session.
+         */
+        if (!ds->use_ice) {
+            pjmedia_transport_info info;
+            pj_sockaddr src_addr;
+            pj_bool_t src_addr_avail = PJ_TRUE;
+
+            pjmedia_transport_get_info(ds->srtp->member_tp, &info);
+            if (idx == RTP_CHANNEL) {
+                if (!pj_sockaddr_has_addr(&info.src_rtp_name)) {
+                    src_addr_avail = PJ_FALSE;
+                } else {
+                    pj_sockaddr_cp(&src_addr, &info.src_rtp_name);
```

Comment:
This is RTP address, not SDP address.

Reply:
> This is RTP address, not SDP address.

Yes, it is to get the message's source address. 
The SDP address/check address is here:
https://github.com/pjsip/pjproject/blob/dcf9c6505738023ff821f38d6eb4e3695fc49d24/pjmedia/src/pjmedia/transport_srtp_dtls.c#L1181-L1191